src/run.js

```diff
@@ -43,7 +43,7 @@ async function getOptions (argv) {
     .option('--sort-commits [property]', `sort commits by property, default: ${DEFAULT_OPTIONS.sortCommits}`)
     .option('--include-branch [branch]', 'one or more branches to include commits from, comma separated', str => str.split(','))
     .option('--release-summary', 'use tagged commit message body as release summary')
-    .option('--handlebars-setup', 'handlebars setup file')
+    .option('--handlebars-setup <file>', 'handlebars setup file')
```

Comment:
Can we use `[file]` to be consistent with the above?
```suggestion
    .option('--handlebars-setup [file]', 'handlebars setup file')
```

Reply:
That was my first thought, but according to [commander](https://github.com/tj/commander.js#other-option-types-negatable-boolean-and-flagvalue) doku the arguments in the square brackets are optional. `--handlebars-setup` option currently does not make sense without value (without JS-file). Therefore I think it would be cleaner if the option value for `--handlebars-setup` would be mandatory (pointy brackets). If this option value is missing, then a CLI compliant and clear exception is thrown.

Reply:
Woah, I never realised this. I'm happy to merge this, and I'll fix up the others.